astro/src/content/docs/lifecycle/manage-users/verification/_common-questions-gating.mdx

```diff
@@ -2,9 +2,9 @@
 ---
```

Comment:
I'm unsure about keeping shared common questions here. They can't search for unverified phone users as of now.

Reply:
Added in 460270abec63a2aa6bc3de205b997ae299040ed8